### PR TITLE
[UX] Show location accuracy on map

### LINKED ISSUE #979: [Map] Show GPS metadata on map
Some ideas (and/or):

* Add blue circle around blue dot depicting current GPS accuracy
* Display number of satellites and accuracy overlaid on map (where? @coreyleamon could you advise?)

### PR DESCRIPTION
**Pull request by shobhitagarwal1612**
Fixes #979

 * Adds an overlay on map with location accuracy
 * GPS satellite updates can't be fetched using current `FusedLocationProviderClient`. Needs more effort.

@gino-m  PTAL?

### CODE REVIEW

File: gnd/src/main/res/layout/map_container_frag.xml [hunk 1]
```diff
@@ -64,6 +64,23 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center|bottom"
+        android:layout_margin="10dp">
+
+        <TextView
+          android:id="@+id/location_accuracy"
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:background="@color/colorAccent"
```
Comment (gino-m) [inline gnd/src/main/res/layout/map_container_frag.xml:78]: Maybe white background, primary color text?

**Review by gino-m — CHANGES_REQUESTED** [review]
Thanks for implementing this much needed feature. Could you add a screenshot to the PR description so we can tweak the UI before merging?

File: gnd/src/main/res/layout/map_container_frag.xml (hunk 1 above)
Comment (shobhitagarwal1612) [reply gnd/src/main/res/layout/map_container_frag.xml]: Done

**Comment by shobhitagarwal1612** [comment]
image

**Comment by shobhitagarwal1612** [comment]
@gino-m Does it look ok?

**Review by gino-m — APPROVED** [review]
Sorry for the delay. LGTM!

I think in the future we may want to ask for UX help (@coreyleamon, @jacobmclaws ) on a more robust way to represent location accuracy, since the text may be long in other languages, and may not be obvious to non-technical folks. That said, for a first pass this works great!

**Pull request merged**